src/App.tsx

```diff
@@ -1,25 +1,120 @@
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list

Reply:
The logic is - to hide warning messages only if select and title are not empty, if you press 'add' messages change color for 1 sec, actually I need setTimeout especially for this purpose ) It seemed to me more interesting but I can rewrite strictly according to the task)

Reply:
fixed